improver/calibration/simple_bias_correction.py

```diff
@@ -363,35 +406,34 @@ def process(
         in the masked areas.
 
         Args:
-            forecast:
-                The cube to which bias correction is to be applied.
-            bias:
-                The cubelist containing the bias values for which to use in
-                the bias correction.
-            lower_bound:
-                A lower bound below which all values will be remapped to
-                after the bias correction step.
-            upper_bound:
-                An upper bound above which all values will be remapped to
-                after the bias correction step.
-            fill_masked_bias_values:
-                Flag to specify whether masked areas in the bias data
-                should be filled to an appropriate fill value.
+            cubes:
+                A list of cubes containing:
+                - A Cube containing the forecast to be calibrated. The input format is expected
+                to be realizations.
+                - A cube or cubelist containing forecast bias data over a specified
+                set of forecast reference times. If a list of cubes is passed in, each cube
+                should represent the forecast error for a single forecast reference time; the
+                mean value will then be evaluated over the forecast_reference_time coordinate.
```

Comment:
I think "a cube or cubelist" isn't so meaningful here.  All positional arguments can be a cube or cubelist so I'm not sure what this means.
Perhaps instead say "a single forecast bias data cube over a specified set of forecast reference times can be provided or otherwise a forecast error cube corresponding to each single forecast reference time..."

Reply:
I'm not sure I agree on this one, the function can't parse  a forecast CubeList or a bias_data Cube. Its purpose is to check whether/that only a single forecast_reference_time/forecast_period is provided. How strongly do you think this should be updated?

Reply:
> How strongly do you think this should be updated?

Was just a suggestion 👍